<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSourceUpdateWidget</class>
 <widget class="QDialog" name="MusicSourceUpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>软件升级精灵</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>454</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FCFCFC;</string>
   </property>
   <widget class="QWidget" name="body" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>454</width>
      <height>290</height>
     </rect>
    </property>
    <widget class="QLabel" name="titleLable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>454</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px; color:rgb(100, 100, 100);</string>
     </property>
     <property name="text">
      <string>天天酷音正在检测版本号...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upgradeButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>253</y>
       <width>88</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>立即升级</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>253</y>
       <width>88</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-radius:3px;background-color:#FCFCFC;color:#888888;font-size:13px;}
QPushButton::hover{background-color:#DDDDDD;color:#444444;}</string>
     </property>
     <property name="text">
      <string>以后再说</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>245</y>
       <width>454</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#F7F7F7;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>titleLable</zorder>
    <zorder>upgradeButton</zorder>
    <zorder>cancelButton</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>90</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/toolSets/lb_upgrade_main</pixmap>
   </property>
  </widget>
  <zorder>container</zorder>
  <zorder>background</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <resources>
  <include location="../../../../TTKUi/TTKModule.qrc"/>
 </resources>
 <connections/>
</ui>
